.github/workflows/coverage.yml

```diff
@@ -49,6 +50,22 @@ jobs:
           cmake ..
           make && sudo make install
 
+      - name: Setup MUSIC
+        run: |
+          python3 -m venv music-venv
+          source music-venv/bin/activate
+          python3 -m pip install mpi4py cython numpy
+          sudo mkdir -p $MUSIC_INSTALL_DIR
+          sudo chown -R $USER $MUSIC_INSTALL_DIR
+          # Stable build: https://github.com/INCF/MUSIC/archive/refs/heads/switch-to-MPI-C-interface.zip  @ f33b66ea9348888eed1761738ab48c23ffc8a0d0
```

Comment:
@mdjurfeldt what are the current plans for merging that and creating a release?

Reply:
@alexsavulescu I'll do it ASAP (hopefully during this weekend) and then tell you in this PR.

Reply:
@alexsavulescu MUSIC 1.2.0 has been release.